libs/designsystem/src/lib/components/local-navigation-item-poc/local-navigation-item-poc.component.html

```diff
@@ -0,0 +1 @@
+<div class="tab-item-inner"><ng-content></ng-content></div>
```

Comment:
```suggestion
<button class="tab-item-inner"><span><ng-content></ng-content></span></button>
```
Doing something like this would give us all the nice basics from buttons (that need to be styled to look correctly, of course), and the wrapper span allows us to do styling without `ng-deep` as far as I can see.

This way consumers dont need to wrap stuff in spans either